resource-detection-gcp/README.md

```diff
@@ -0,0 +1,67 @@
+# Resource Detection Example - GCP
+
+> [!NOTE]
+> Running this example does not generate any telemetry. This example merely demonstrates resource attributes detetcted by the GCP Resource Detector.
```

Comment:
instead, maybe emit a single span from `main`?

Reply:
I think the original intent behind this example was to show what attributes are detected by this particular detector. Essentially, highlight the purpose of a resource detector in isolation.

The note here was to make it clear to the user not expect any telemetry since most examples would generate some kind of telemetry. 

Does this make sense ?

Reply:
I was thinking if you emit a single span from your example in `main`, then you don't need this note, and users could see the resource attributes that were attached to that span

Reply:
Sounds good, thanks for the suggestion !